### PR TITLE
pythonPackages.tableaudocumentapi: init at 0.6

### PR DESCRIPTION
**Pull request by costrouc**
###### Motivation for this change

Working with tableau and want to experiment with python package for dealing with tableau documents.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by costrouc** [comment]
@GrahamcOfBorg build python2Packages.tableaudocumentapi python3Packages.tableaudocumentapi

File: pkgs/development/python-modules/tableaudocumentapi/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "fc6d44b62cf6ea29916c073686e2f9f35c9902eccd57b8493f8d44a59a2f60d9";
+  };
+
+  # tests not inclued with release
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A Python module for working with Tableau files.";
```
Comment (dotlambda) [inline pkgs/development/python-modules/tableaudocumentapi/default.nix]:
```suggestion
    description = "A Python module for working with Tableau files";
```
See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

Comment (costrouc) [reply pkgs/development/python-modules/tableaudocumentapi/default.nix]: Fixed thanks for catching that.

**Comment by worldofpeace** [comment]
I noticing that the repo is called `document-api-python` and on PyPi it's `tableaudocumentapi`.

Not sure what will be more discoverable but the hyphenated one is more readable.

**Comment by dotlambda** [comment]
We usually go with the PyPI name.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.tableaudocumentapi python3.pkgs.tableaudocumentapi

**Pull request merged**